{{cookiecutter.project_slug}}/Makefile

```diff
@@ -115,7 +115,7 @@ connect-training:  ### Execute shell to the training job
 	neuro exec --no-key-check $(TRAINING_NAME) bash
 
 .PHONY: jupyter
-jupyter:  ### Run jupyter job
+jupyter:  upload_code upload_notebooks ### Run jupyter job
```

Comment:
The targets are already renamed. They use `-` instead of `_`.

Reply:
Blind copy-paste from issues = evil :(
Thanks for this catch!